function animate(el, animation) {
  el = $(el);
  el.addClass(animation + ' animated').one('webkitAnimationEnd mozAnimationEnd MSAnimationEnd oanimationend animationend', function () {
    el.removeClass(animation);
    el.removeClass('animated');
  });
}


var allTools = [{
  "elems": [{
    "panel_section_name": "Get Data",
    "description": "from your computer",
    "panel_section_id": "getext",
    "version": "1.1.4",
    "link": "/tool_runner?tool_id=upload1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "upload1",
    "name": "Upload File"
  }, {
    "panel_section_name": "Get Data",
    "description": "table browser",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=ucsc_table_direct1",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "ucsc_table_direct1",
    "name": "UCSC Main"
  }, {
    "panel_section_name": "Get Data",
    "description": "table browser",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=ucsc_table_direct_archaea1",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "ucsc_table_direct_archaea1",
    "name": "UCSC Archaea"
  }, {
    "panel_section_name": "Get Data",
    "description": "ENA SRA",
    "panel_section_id": "getext",
    "version": "1.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=ebi_sra_main",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "ebi_sra_main",
    "name": "EBI SRA"
  }, {
    "panel_section_name": "Get Data",
    "description": "Central server",
    "panel_section_id": "getext",
    "version": "1.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=biomart",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "biomart",
    "name": "BioMart"
  }, {
    "panel_section_name": "Get Data",
    "description": "Central server",
    "panel_section_id": "getext",
    "version": "1.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=gramenemart",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "gramenemart",
    "name": "GrameneMart"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=flymine",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "flymine",
    "name": "Flymine"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=modENCODEfly",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "modENCODEfly",
    "name": "modENCODE fly"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=modmine",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "modmine",
    "name": "modENCODE modMine"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=mousemine",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "mousemine",
    "name": "MouseMine"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=ratmine",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "ratmine",
    "name": "Ratmine"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=yeastmine",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "yeastmine",
    "name": "YeastMine"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=modENCODEworm",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "modENCODEworm",
    "name": "modENCODE worm"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=wormbase",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "wormbase",
    "name": "WormBase"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=zebrafishmine",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "zebrafishmine",
    "name": "ZebrafishMine"
  }, {
    "panel_section_name": "Get Data",
    "description": "server",
    "panel_section_id": "getext",
    "version": "1.0.0",
    "link": "/tool_runner/data_source_redirect?tool_id=eupathdb",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "eupathdb",
    "name": "EuPathDB"
  }, {
    "panel_section_name": "Get Data",
    "description": "from file browser",
    "panel_section_id": "getext",
    "version": "0.0.1",
    "link": "/tool_runner/data_source_redirect?tool_id=genomespace_file_browser_prod",
    "target": "_top",
    "min_width": "800",
    "model_class": "DataSourceTool",
    "id": "genomespace_file_browser_prod",
    "name": "GenomeSpace import"
  }], "model_class": "ToolSection", "version": "", "id": "getext", "name": "Get Data"
}, {
  "elems": [{
    "panel_section_name": "Lift-Over",
    "description": "between assemblies and genomes",
    "panel_section_id": "liftOver",
    "version": "1.0.3",
    "link": "/tool_runner?tool_id=liftOver1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "liftOver1",
    "name": "Convert genome coordinates"
  }], "model_class": "ToolSection", "version": "", "id": "liftOver", "name": "Lift-Over"
}, {
  "elems": [{
    "panel_section_name": "Text Manipulation",
    "description": "an expression on every row",
    "panel_section_id": "textutil",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcolumn_maker%2FAdd_a_column1%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.1.0",
    "name": "Compute"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "to an existing dataset",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=addValue",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "addValue",
    "name": "Add column"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "tail-to-head",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=cat1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "cat1",
    "name": "Concatenate datasets"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "consecutive characters",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Condense+characters1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Condense characters1",
    "name": "Condense"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "delimiters to TAB",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Convert+characters1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Convert characters1",
    "name": "Convert"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "together",
    "panel_section_id": "textutil",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=mergeCols1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "mergeCols1",
    "name": "Merge Columns"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "as a new dataset",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=createInterval",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "createInterval",
    "name": "Create single interval"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "columns from a table",
    "panel_section_id": "textutil",
    "version": "1.0.2",
    "link": "/tool_runner?tool_id=Cut1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Cut1",
    "name": "Cut"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "of selected columns",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=ChangeCase",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "ChangeCase",
    "name": "Change Case"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "two files side by side",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Paste1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Paste1",
    "name": "Paste"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "of a file",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Remove+beginning1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Remove beginning1",
    "name": "Remove beginning"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "from a file",
    "panel_section_id": "textutil",
    "version": "2.0.1",
    "link": "/tool_runner?tool_id=random_lines1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "random_lines1",
    "name": "Select random lines"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "lines from a dataset",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Show+beginning1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Show beginning1",
    "name": "Select first"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "lines from a dataset",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Show+tail1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Show tail1",
    "name": "Select last"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "leading or trailing characters",
    "panel_section_id": "textutil",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=trimmer",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "trimmer",
    "name": "Trim"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "of a dataset",
    "panel_section_id": "textutil",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=wc_gnu",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "wc_gnu",
    "name": "Line/Word/Character count"
  }, {
    "panel_section_name": "Text Manipulation",
    "description": "on a dataset",
    "panel_section_id": "textutil",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=secure_hash_message_digest",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "secure_hash_message_digest",
    "name": "Secure Hash / Message Digest"
  }], "model_class": "ToolSection", "version": "", "id": "textutil", "name": "Text Manipulation"
}, {
  "elems": [{
    "panel_section_name": "FASTA manipulation",
    "description": "converts tabular file to FASTA format",
    "panel_section_id": "fasta_manipulation",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftabular_to_fasta%2Ftab2fasta%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/tabular_to_fasta/tab2fasta/1.1.0",
    "name": "Tabular-to-FASTA"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "converter",
    "panel_section_id": "fasta_manipulation",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_to_tabular%2Ffasta2tab%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_to_tabular/fasta2tab/1.1.0",
    "name": "FASTA-to-Tabular"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "converter",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_to_fasta%2Fcshl_fastq_to_fasta%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_to_fasta/cshl_fastq_to_fasta/1.0.0",
    "name": "FASTQ to FASTA"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "converter",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqtofasta%2Ffastq_to_fasta_python%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastqtofasta/fastq_to_fasta_python/1.0.0",
    "name": "FASTQ to FASTA"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "converter",
    "panel_section_id": "convert",
    "version": "2.0.0",
    "link": "/tool_runner?tool_id=bed2gff1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "bed2gff1",
    "name": "BED-to-GFF"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "converter",
    "panel_section_id": "convert",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=gff2bed1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "gff2bed1",
    "name": "GFF-to-BED"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "Converts a MAF formatted file to the BED format",
    "panel_section_id": "convert",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=MAF_To_BED1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_To_BED1",
    "name": "MAF to BED"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "Converts a MAF formatted file to the Interval format",
    "panel_section_id": "convert",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=MAF_To_Interval1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_To_Interval1",
    "name": "MAF to Interval"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "Converts a MAF formatted file to FASTA format",
    "panel_section_id": "convert",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=MAF_To_Fasta1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_To_Fasta1",
    "name": "MAF to FASTA"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "",
    "panel_section_id": "convert",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=Sff_extractor",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Sff_extractor",
    "name": "SFF converter"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "converter",
    "panel_section_id": "convert",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=wig_to_bigWig",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "wig_to_bigWig",
    "name": "Wig/BedGraph-to-bigWig"
  }, {
    "panel_section_name": "Convert Formats",
    "description": "converter",
    "panel_section_id": "convert",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=bed_to_bigBed",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "bed_to_bigBed",
    "name": "BED-to-bigBed"
  }], "model_class": "ToolSection", "version": "", "id": "convert", "name": "Convert Formats"
}, {
  "elems": [{
    "panel_section_name": "Filter and Sort",
    "description": "of a numeric column",
    "panel_section_id": "filter",
    "version": "1.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdna_filtering%2Fhistogram_rpy%2F1.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/dna_filtering/histogram_rpy/1.0.3",
    "name": "Histogram"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "data on any column using simple expressions",
    "panel_section_id": "filter",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=Filter1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Filter1",
    "name": "Filter"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "data in ascending or descending order",
    "panel_section_id": "filter",
    "version": "1.0.3",
    "link": "/tool_runner?tool_id=sort1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "sort1",
    "name": "Sort"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "lines that match an expression",
    "panel_section_id": "filter",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=Grep1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Grep1",
    "name": "Select"
  }, {
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gff",
    "text": "GFF"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "from GFF data",
    "panel_section_id": "filter",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=Extract_features1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Extract_features1",
    "name": "Extract features"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "using simple expressions",
    "panel_section_id": "filter",
    "version": "0.1",
    "link": "/tool_runner?tool_id=gff_filter_by_attribute",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "gff_filter_by_attribute",
    "name": "Filter GFF data by attribute"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "using simple expressions",
    "panel_section_id": "filter",
    "version": "0.1",
    "link": "/tool_runner?tool_id=gff_filter_by_feature_count",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "gff_filter_by_feature_count",
    "name": "Filter GFF data by feature count"
  }, {
    "panel_section_name": "Filter and Sort",
    "description": "",
    "panel_section_id": "filter",
    "version": "0.1",
    "link": "/tool_runner?tool_id=gtf_filter_by_attribute_values_list",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "gtf_filter_by_attribute_values_list",
    "name": "Filter GTF data by attribute values_list"
  }], "model_class": "ToolSection", "version": "", "id": "filter", "name": "Filter and Sort"
}, {
  "elems": [{
    "panel_section_name": "Join, Subtract and Group",
    "description": "from another dataset",
    "panel_section_id": "group",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsubtract_query%2Fsubtract_query1%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/subtract_query/subtract_query1/0.1",
    "name": "Subtract Whole Dataset"
  }, {
    "panel_section_name": "Join, Subtract and Group",
    "description": "side by side on a specified field",
    "panel_section_id": "group",
    "version": "2.0.2",
    "link": "/tool_runner?tool_id=join1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "join1",
    "name": "Join two Datasets"
  }, {
    "panel_section_name": "Join, Subtract and Group",
    "description": "to find common or distinct rows",
    "panel_section_id": "group",
    "version": "1.0.2",
    "link": "/tool_runner?tool_id=comp1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "comp1",
    "name": "Compare two Datasets"
  }, {
    "panel_section_name": "Join, Subtract and Group",
    "description": "data by a column and perform aggregate operation on other columns.",
    "panel_section_id": "group",
    "version": "2.1.0",
    "link": "/tool_runner?tool_id=Grouping1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Grouping1",
    "name": "Group"
  }], "model_class": "ToolSection", "version": "", "id": "group", "name": "Join, Subtract and Group"
}, {
  "elems": [{
    "panel_section_name": "NGS: QC and manipulation",
    "description": "Read Quality reports",
    "panel_section_id": "cshl_library_information",
    "version": "0.63",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqc%2Ffastqc%2F0.63",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.63",
    "name": "FastQC"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fshort_reads_trim_seq%2Ftrim_reads%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/short_reads_trim_seq/trim_reads/1.0.0",
    "name": "Select high quality segments"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fshort_reads_figure_score%2Fquality_score_distribution%2F1.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/short_reads_figure_score/quality_score_distribution/1.0.2",
    "name": "Build base quality distribution"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_boxplot%2Fcshl_fastq_quality_boxplot%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_quality_boxplot/cshl_fastq_quality_boxplot/1.0.0",
    "name": "Draw quality score boxplot"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "(ASCII-Numeric)",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_converter%2Fcshl_fastq_quality_converter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_quality_converter/cshl_fastq_quality_converter/1.0.0",
    "name": "Quality format converter"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_filter%2Fcshl_fastq_quality_filter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_quality_filter/cshl_fastq_quality_filter/1.0.0",
    "name": "Filter by quality"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "converter",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_to_fasta%2Fcshl_fastq_to_fasta%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_to_fasta/cshl_fastq_to_fasta/1.0.0",
    "name": "FASTQ to FASTA"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_artifacts_filter%2Fcshl_fastx_artifacts_filter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_artifacts_filter/cshl_fastx_artifacts_filter/1.0.0",
    "name": "Remove sequencing artifacts"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_barcode_splitter%2Fcshl_fastx_barcode_splitter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_barcode_splitter/cshl_fastx_barcode_splitter/1.0.0",
    "name": "Barcode Splitter"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "adapter sequences",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_clipper%2Fcshl_fastx_clipper%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_clipper/cshl_fastx_clipper/1.0.1",
    "name": "Clip"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "sequences",
    "panel_section_id": "fasta_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_collapser%2Fcshl_fastx_collapser%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_collapser/cshl_fastx_collapser/1.0.0",
    "name": "Collapse"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_nucleotides_distribution%2Fcshl_fastx_nucleotides_distribution%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_nucleotides_distribution/cshl_fastx_nucleotides_distribution/1.0.0",
    "name": "Draw nucleotides distribution chart"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_quality_statistics%2Fcshl_fastx_quality_statistics%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_quality_statistics/cshl_fastx_quality_statistics/1.0.0",
    "name": "Compute quality statistics"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "0.0.11",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_renamer%2Fcshl_fastx_renamer%2F0.0.11",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_renamer/cshl_fastx_renamer/0.0.11",
    "name": "Rename sequences"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_reverse_complement%2Fcshl_fastx_reverse_complement%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_reverse_complement/cshl_fastx_reverse_complement/1.0.0",
    "name": "Reverse-Complement"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_trimmer%2Fcshl_fastx_trimmer%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_trimmer/cshl_fastx_trimmer/1.0.0",
    "name": "Trim sequences"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "into FASTQ",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_combiner%2Ffastq_combiner%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_combiner/fastq_combiner/1.0.1",
    "name": "Combine FASTA and QUAL"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "reads by quality score and length",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_filter%2Ffastq_filter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_filter/fastq_filter/1.0.0",
    "name": "Filter FASTQ"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "reads on various attributes",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_manipulation%2Ffastq_manipulation%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_manipulation/fastq_manipulation/1.0.1",
    "name": "Manipulate FASTQ"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "convert between various FASTQ quality formats",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_groomer%2Ffastq_groomer%2F1.0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_groomer/fastq_groomer/1.0.4",
    "name": "FASTQ Groomer"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "by quality score",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_masker_by_quality%2Ffastq_masker_by_quality%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_masker_by_quality/fastq_masker_by_quality/1.0.0",
    "name": "FASTQ Masker"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "on paired end reads",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_paired_end_joiner%2Ffastq_paired_end_joiner%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_paired_end_joiner/fastq_paired_end_joiner/1.0.0",
    "name": "FASTQ joiner"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "on joined paired end reads",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_paired_end_splitter%2Ffastq_paired_end_splitter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_paired_end_splitter/fastq_paired_end_splitter/1.0.0",
    "name": "FASTQ splitter"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "by column",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_stats%2Ffastq_stats%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_stats/fastq_stats/1.0.0",
    "name": "FASTQ Summary Statistics"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "converter",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqtofasta%2Ffastq_to_fasta_python%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastqtofasta/fastq_to_fasta_python/1.0.0",
    "name": "FASTQ to FASTA"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "converter",
    "panel_section_id": "cshl_library_information",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_to_tabular%2Ffastq_to_tabular%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_to_tabular/fastq_to_tabular/1.1.0",
    "name": "FASTQ to Tabular"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "by column",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_trimmer%2Ffastq_trimmer%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_trimmer/fastq_trimmer/1.0.0",
    "name": "FASTQ Trimmer"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "by sliding window",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_trimmer_by_quality%2Ffastq_quality_trimmer%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastq_trimmer_by_quality/fastq_quality_trimmer/1.0.0",
    "name": "FASTQ Quality Trimmer"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "converter",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftabular_to_fastq%2Ftabular_to_fastq%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/tabular_to_fastq/tabular_to_fastq/1.0.0",
    "name": "Tabular to FASTQ"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "SOLiD output to fastq",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=solid2fastq",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "solid2fastq",
    "name": "Convert"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "for SOLiD data",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=solid_qual_stats",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "solid_qual_stats",
    "name": "Compute quality statistics"
  }, {
    "panel_section_name": "NGS: QC and manipulation",
    "description": "for SOLiD data",
    "panel_section_id": "cshl_library_information",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=solid_qual_boxplot",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "solid_qual_boxplot",
    "name": "Draw quality score boxplot"
  }], "model_class": "ToolSection", "version": "", "id": "cshl_library_information", "name": "NGS: QC and manipulation"
}, {
  "elems": [{
    "panel_section_name": "NGS: Mapping",
    "description": "- map reads against reference genome",
    "panel_section_id": "ngs_mapping",
    "version": "0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbowtie2%2Fbowtie2%2F0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2/0.4",
    "name": "Bowtie2"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "- map short reads (< 100 bp) against reference genome",
    "panel_section_id": "ngs_mapping",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbwa%2Fbwa%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa/0.1",
    "name": "BWA"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "- map medium and long reads (> 100 bp) against reference genome",
    "panel_section_id": "ngs_mapping",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbwa%2Fbwa_mem%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/0.1",
    "name": "BWA-MEM"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "",
    "panel_section_id": "ngs_mapping",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmegablast_xml_parser%2Fmegablast_xml_parser%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/megablast_xml_parser/megablast_xml_parser/1.0.0",
    "name": "Parse blast XML output"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "compare short reads against htgs, nt, and wgs databases",
    "panel_section_id": "ngs_mapping",
    "version": "1.2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmegablast_wrapper%2Fmegablast_wrapper%2F1.2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/megablast_wrapper/megablast_wrapper/1.2.0",
    "name": "Megablast"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "",
    "panel_section_id": "ngs_mapping",
    "version": "1.2.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbwa_wrappers%2Fbwa_wrapper%2F1.2.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bwa_wrappers/bwa_wrapper/1.2.3",
    "name": "Map with BWA for Illumina"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "",
    "panel_section_id": "ngs_mapping",
    "version": "1.1.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbowtie_wrappers%2Fbowtie_wrapper%2F1.1.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bowtie_wrappers/bowtie_wrapper/1.1.2",
    "name": "Map with Bowtie for Illumina"
  }, {
    "panel_section_name": "NGS: Mapping",
    "description": "map short reads against reference sequence",
    "panel_section_id": "ngs_mapping",
    "version": "1.2.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flastz%2Flastz_wrapper_2%2F1.2.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/lastz/lastz_wrapper_2/1.2.2",
    "name": "Lastz"
  }], "model_class": "ToolSection", "version": "", "id": "ngs_mapping", "name": "NGS: Mapping"
}, {
  "elems": [{
    "panel_section_name": "NGS: RNA-seq",
    "description": "find significant changes in transcript expression, splicing, and promoter use",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffdiff%2Fcuffdiff%2F2.2.1.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cuffdiff/cuffdiff/2.2.1.2",
    "name": "Cuffdiff"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "Gapped-read mapper for RNA-seq data",
    "panel_section_id": "ngs-rna-tools",
    "version": "0.9",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftophat2%2Ftophat2%2F0.9",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/tophat2/tophat2/0.9",
    "name": "Tophat"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "transcript assembly and quantification",
    "panel_section_id": "ngs-rna-tools",
    "version": "1.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstringtie%2Fstringtie%2F1.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/iuc/stringtie/stringtie/1.0.3",
    "name": "StringTie"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "compare assembled transcripts to a reference annotation and track Cufflinks transcripts across multiple experiments",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffcompare%2Fcuffcompare%2F2.2.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cuffcompare/cuffcompare/2.2.1.0",
    "name": "Cuffcompare"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "transcript assembly and FPKM (RPKM) estimates for RNA-Seq data",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcufflinks%2Fcufflinks%2F2.2.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cufflinks/cufflinks/2.2.1.0",
    "name": "Cufflinks"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "Filters and/or converts GFF3/GTF2 records",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgffread%2Fgffread%2F2.2.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/gffread/gffread/2.2.1.0",
    "name": "gffread"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "merge together several Cufflinks assemblies",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffmerge%2Fcuffmerge%2F2.2.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cuffmerge/cuffmerge/2.2.1.0",
    "name": "Cuffmerge"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "Create normalized expression levels",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffnorm%2Fcuffnorm%2F2.2.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cuffnorm/cuffnorm/2.2.1.0",
    "name": "Cuffnorm"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "Precompute gene expression levels",
    "panel_section_id": "ngs-rna-tools",
    "version": "2.2.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffquant%2Fcuffquant%2F2.2.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cuffquant/cuffquant/2.2.1.0",
    "name": "Cuffquant"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "using tracking file",
    "panel_section_id": "ngs-rna-tools",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffilter_transcripts_via_tracking%2Ffilter_combined_via_tracking%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/filter_transcripts_via_tracking/filter_combined_via_tracking/0.1",
    "name": "Filter Combined Transcripts"
  }, {
    "panel_section_name": "NGS: RNA-seq",
    "description": "post-processing to identify fusion genes",
    "panel_section_id": "ngs-rna-tools",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftophat_fusion_post%2Ftophat_fusion_post%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/tophat_fusion_post/tophat_fusion_post/0.1",
    "name": "Tophat Fusion Post"
  }], "model_class": "ToolSection", "version": "", "id": "ngs-rna-tools", "name": "NGS: RNA-seq"
}, {
  "elems": [{
    "panel_section_name": "NGS: SAMtools",
    "description": "Converts BCF format to VCF format",
    "panel_section_id": "samtools",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgregory-minevich%2Fbcftools_view%2Fbcftools_view%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/gregory-minevich/bcftools_view/bcftools_view/0.0.1",
    "name": "bcftools view"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "copy SAM/BAM header between datasets",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_reheader%2Fsamtools_reheader%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_reheader/samtools_reheader/2.0",
    "name": "Reheader"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "BAM dataset on readgroups",
    "panel_section_id": "samtools",
    "version": "1.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_split%2Fsamtools_split%2F1.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_split/samtools_split/1.1",
    "name": "Split"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "generate statistics for BAM dataset",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_stats%2Fsamtools_stats%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_stats/samtools_stats/2.0",
    "name": "Stats"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "convert BAM to SAM",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbam_to_sam%2Fbam_to_sam%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bam_to_sam/bam_to_sam/2.0",
    "name": "BAM-to-SAM"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "BAM dataset",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_sort%2Fsamtools_sort%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_sort/samtools_sort/2.0",
    "name": "Sort"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "recalculate MD/NM tags",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_calmd%2Fsamtools_calmd%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_calmd/samtools_calmd/2.0",
    "name": "CalMD"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "calculate read depth for a set of genomic intervals",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_bedcov%2Fsamtools_bedcov%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_bedcov/samtools_bedcov/2.0",
    "name": "BedCov"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "tabulate mapping statistics for BAM dataset",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_idxstats%2Fsamtools_idxstats%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_idxstats/samtools_idxstats/2.0",
    "name": "IdxStats"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "tabulate descriptive stats for BAM datset",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_flagstat%2Fsamtools_flagstat%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_flagstat/samtools_flagstat/2.0",
    "name": "Flagstat"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "BAM by genomic regions",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_slice_bam%2Fsamtools_slice_bam%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_slice_bam/samtools_slice_bam/2.0",
    "name": "Slice"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "convert SAM to BAM",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam_to_bam%2Fsam_to_bam%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/sam_to_bam/sam_to_bam/2.0",
    "name": "SAM-to-BAM"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "remove PCR duplicates",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_rmdup%2Fsamtools_rmdup%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_rmdup/samtools_rmdup/2.0",
    "name": "RmDup"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "call variants",
    "panel_section_id": "samtools",
    "version": "2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_mpileup%2Fsamtools_mpileup%2F2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtools_mpileup/samtools_mpileup/2.0",
    "name": "MPileup"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "condenses pileup format into ranges of bases",
    "panel_section_id": "samtools",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpileup_interval%2Fpileup_interval%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/pileup_interval/pileup_interval/1.0.0",
    "name": "Pileup-to-Interval"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "on coverage and SNPs",
    "panel_section_id": "samtools",
    "version": "1.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpileup_parser%2Fpileup_parser%2F1.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/pileup_parser/pileup_parser/1.0.2",
    "name": "Filter pileup"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "to interval",
    "panel_section_id": "samtools",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam2interval%2Fsam2interval%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/sam2interval/sam2interval/1.0.1",
    "name": "Convert SAM"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "on bitwise flag values",
    "panel_section_id": "samtools",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam_bitwise_flag_filter%2Fsam_bw_filter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/sam_bitwise_flag_filter/sam_bw_filter/1.0.0",
    "name": "Filter SAM"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "from BAM dataset",
    "panel_section_id": "samtools",
    "version": "1.1.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam_pileup%2Fsam_pileup%2F1.1.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/sam_pileup/sam_pileup/1.1.2",
    "name": "Generate pileup"
  }, {
    "panel_section_name": "NGS: SAMtools",
    "description": "files on FLAG MAPQ RG LN or by region",
    "panel_section_id": "samtools",
    "version": "1.1.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtool_filter2%2Fsamtool_filter2%2F1.1.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/samtool_filter2/samtool_filter2/1.1.1",
    "name": "Filter SAM or BAM, output SAM or BAM"
  }], "model_class": "ToolSection", "version": "", "id": "samtools", "name": "NGS: SAMtools"
}, {
  "elems": [{
    "panel_section_name": "NGS: BAM Tools",
    "description": "BAM datasets and perform other transformations",
    "panel_section_id": "ngs:_bam_tools",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbamtools%2Fbamtools%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bamtools/bamtools/0.0.1",
    "name": "Convert, Merge, Randomize"
  }, {
    "panel_section_name": "NGS: BAM Tools",
    "description": "BAM datasets on a variety of attributes",
    "panel_section_id": "ngs:_bam_tools",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbamtools_filter%2FbamFilter%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bamtools_filter/bamFilter/0.0.1",
    "name": "Filter"
  }, {
    "panel_section_name": "NGS: BAM Tools",
    "description": "BAM datasets on variety of attributes",
    "panel_section_id": "ngs:_bam_tools",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbamtools_split%2FbamSplit%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/bamtools_split/bamSplit/0.0.1",
    "name": "Split"
  }], "model_class": "ToolSection", "version": "", "id": "ngs:_bam_tools", "name": "NGS: BAM Tools"
}, {
  "elems": [{
    "panel_section_name": "NGS: Picard",
    "description": "replace header in a SAM/BAM dataset",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ReplaceSamHeader%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_ReplaceSamHeader/1.126.0",
    "name": "ReplaceSamHeader"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "add comments to BAM dataset",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_AddCommentsToBam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_AddCommentsToBam/1.126.0",
    "name": "AddCommentsToBam"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "extract reads and qualities from SAM/BAM dataset and convert to fastq",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_SamToFastq%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_SamToFastq/1.126.0",
    "name": "SamToFastq"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "Downsample a file to retain a subset of the reads",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_DownsampleSam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_DownsampleSam/1.126.0",
    "name": "Downsample SAM/BAM"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "examine aligned records in BAM datasets to locate duplicate molecules",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MarkDuplicates%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/1.126.0",
    "name": "MarkDuplicates"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "reorder reads to match ordering in reference sequences",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ReorderSam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_ReorderSam/1.126.0",
    "name": "ReorderSam"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "revert SAM/BAM datasets to a previous state",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_RevertSam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_RevertSam/1.126.0",
    "name": "RevertSam"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "assess sequence library complexity from read sequences",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_EstimateLibraryComplexity%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_EstimateLibraryComplexity/1.126.0",
    "name": "EstimateLibraryComplexity"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "sort SAM/BAM dataset",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_SortSam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_SortSam/1.126.0",
    "name": "SortSam"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "add or replaces read group information",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_AddOrReplaceReadGroups%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_AddOrReplaceReadGroups/1.126.0",
    "name": "AddOrReplaceReadGroups"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "collect metrics about the alignment of RNA to various functional classes of loci in the genome",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectRnaSeqMetrics%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CollectRnaSeqMetrics/1.126.0",
    "name": "CollectRnaSeqMetrics"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "perform SAM/BAM grooming",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CleanSam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CleanSam/1.126.0",
    "name": "CleanSam"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "plots distribution of insert sizes",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectInsertSizeMetrics%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CollectInsertSizeMetrics/1.126.0",
    "name": "CollectInsertSizeMetrics"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "chart distribution of base qualities",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MeanQualityByCycle%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MeanQualityByCycle/1.126.0",
    "name": "MeanQualityByCycle"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "assess validity of SAM/BAM dataset",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ValidateSamFile%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_ValidateSamFile/1.126.0",
    "name": "ValidateSamFile"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "charts the nucleotide distribution per cycle in a SAM or BAM dataset",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectBaseDistributionByCycle%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CollectBaseDistributionByCycle/1.126.0",
    "name": "CollectBaseDistributionByCycle"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "compute metrics for evaluating of whole genome sequencing experiments",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectWgsMetrics%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CollectWgsMetrics/1.126.0",
    "name": "CollectWgsMetrics"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "writes a file containing summary alignment metrics",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CASM%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CASM/1.126.0",
    "name": "Collect Alignment Summary Metrics"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "charts the GC bias metrics",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectGcBiasMetrics%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_CollectGcBiasMetrics/1.126.0",
    "name": "CollectGcBiasMetrics"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "normalize fasta datasets",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_NormalizeFasta%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_NormalizeFasta/1.126.0",
    "name": "NormalizeFasta"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "ensure that all mate-pair information is in sync between each read and it's mate pair",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FixMateInformation%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_FixMateInformation/1.126.0",
    "name": "FixMateInformation"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "merges multiple SAM/BAM datasets into one",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MergeSamFiles%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MergeSamFiles/1.126.0",
    "name": "MergeSamFiles"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "merge alignment data with additional info stored in an unmapped BAM dataset",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MergeBamAlignment%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MergeBamAlignment/1.126.0",
    "name": "MergeBamAlignment"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "revert the original base qualities and add the mate cigar tag",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_RevertOriginalBaseQualitiesAndAddMateCigar%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_RevertOriginalBaseQualitiesAndAddMateCigar/1.126.0",
    "name": "RevertOriginalBaseQualitiesAndAddMateCigar"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "include or exclude aligned and unaligned reads and read lists",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FilterSamReads%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_FilterSamReads/1.126.0",
    "name": "FilterSamReads"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "convert Fastq data into unaligned BAM",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FastqToSam%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_FastqToSam/1.126.0",
    "name": "FastqToSam"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "convert coordinate data into picard interval list format",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_BedToIntervalList%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_BedToIntervalList/1.126.0",
    "name": "BedToIntervalList"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "chart quality score distribution",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_QualityScoreDistribution%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_QualityScoreDistribution/1.126.0",
    "name": "QualityScoreDistribution"
  }, {
    "panel_section_name": "NGS: Picard",
    "description": "examine aligned records in BAM datasets to locate duplicate molecules",
    "panel_section_id": "ngs:_picard",
    "version": "1.126.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MarkDuplicatesWithMateCigar%2F1.126.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicatesWithMateCigar/1.126.0",
    "name": "MarkDuplicatesWithMateCigar"
  }], "model_class": "ToolSection", "version": "", "id": "ngs:_picard", "name": "NGS: Picard"
}, {
  "elems": [{
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Intersect two VCF datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfvcfintersect%2Fvcfvcfintersect%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfvcfintersect/vcfvcfintersect/0.0.2",
    "name": "VCF-VCFintersect:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Convert genotype-based phased alleles into haplotype alleles",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfgeno2haplo%2Fvcfgeno2haplo%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfgeno2haplo/vcfgeno2haplo/0.0.3",
    "name": "VCFgenotype-to-haplotype:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Combine multiple VCF datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfcombine%2Fvcfcombine%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfcombine/vcfcombine/0.0.3",
    "name": "VCFcombine:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "filter VCF data in a variety of attributes",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcffilter%2Fvcffilter2%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcffilter/vcffilter2/0.0.3",
    "name": "VCFfilter:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Convert VCF data into TAB-delimited format",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcf2tsv%2Fvcf2tsv%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcf2tsv/vcf2tsv/0.0.3",
    "name": "VCFtoTab-delimited:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Split alleleic primitives (gaps or mismatches) into multiple VCF lines",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfallelicprimitives%2Fvcfallelicprimitives%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfallelicprimitives/vcfallelicprimitives/0.0.3",
    "name": "VcfAllelicPrimitives:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Break multiple alleles into multiple records, or combine overallpoing alleles into a single record",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfbreakcreatemulti%2Fvcfbreakcreatemulti%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfbreakcreatemulti/vcfbreakcreatemulti/0.0.3",
    "name": "VCFbreakCreateMulti:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Randomly sample sites from VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfrandomsample%2Fvcfrandomsample%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfrandomsample/vcfrandomsample/0.0.3",
    "name": "VCFrandomSample:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Removes multi-allelic sites by picking the most common alternate",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfflatten%2Fvcfflatten2%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfflatten/vcfflatten2/0.0.3",
    "name": "VCFflatten:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Select samples from a VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfselectsamples%2Fvcfselectsamples%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfselectsamples/vcfselectsamples/0.0.3",
    "name": "VCFselectsamples:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Verify that the reference allele matches the reference genome",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfcheck%2Fvcfcheck%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfcheck/vcfcheck/0.0.3",
    "name": "VCFcheck:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Convert numerical representation of genotypes to allelic",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfgenotypes%2Fvcfgenotypes%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfgenotypes/vcfgenotypes/0.0.3",
    "name": "VCFgenotypes:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Intersect two VCF datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfvcfintersect%2Fvcfvcfintersect%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfvcfintersect/vcfvcfintersect/0.0.3",
    "name": "VCF-VCFintersect:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Count the number of heterozygotes and alleles, compute het/hom ratio",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfhethom%2Fvcfhethom%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfhethom/vcfhethom/0.0.3",
    "name": "VCFhetHomAlleles:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Left-align indels and complex variants in VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfleftalign%2Fvcfleftalign%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfleftalign/vcfleftalign/0.0.3",
    "name": "VCFleftAlign:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Adds info fields from the second dataset which are not present in the first dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfaddinfo%2Fvcfaddinfo%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfaddinfo/vcfaddinfo/0.0.3",
    "name": "VCFaddinfo:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Sort VCF dataset by coordinate",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfsort%2Fvcfsort%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfsort/vcfsort/0.0.2",
    "name": "VCFsort:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Extract flanking sequences for each VCF record",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfprimers%2Fvcfprimers%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfprimers/vcfprimers/0.0.3",
    "name": "VCFprimers:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Annotate genotypes in a VCF dataset using genotypes from another VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfannotategenotypes%2Fvcfannotategenotypes%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfannotategenotypes/vcfannotategenotypes/0.0.3",
    "name": "VCFannotateGenotypes:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Count the allele frequencies across alleles present in each record in the VCF file",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcffixup%2Fvcffixup%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcffixup/vcffixup/0.0.3",
    "name": "VCFfixup:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Intersect VCF and BED datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfbedintersect%2Fvcfbedintersect%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfbedintersect/vcfbedintersect/0.0.3",
    "name": "VCF-BEDintersect:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Calculate distance to the nearest variant",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfdistance%2Fvcfdistance%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfdistance/vcfdistance/0.0.3",
    "name": "VCFdistance:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Output records belonging to samples common between two datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfcommonsamples%2Fvcfcommonsamples%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfcommonsamples/vcfcommonsamples/0.0.3",
    "name": "VCFcommonSamples:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Intersect VCF records with BED annotations",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfannotate%2Fvcfannotate%2F0.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcfannotate/vcfannotate/0.0.3",
    "name": "VCFannotate:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Count the allele frequencies across alleles present in each record in the VCF file",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcffixup%2Fvcffixup%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcffixup/vcffixup/0.0.2",
    "name": "VCFfixup:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Extract flanking sequences for each VCF record",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfprimers%2Fvcfprimers%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfprimers/vcfprimers/0.0.2",
    "name": "VCFprimers:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Combine multiple VCF datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfcombine%2Fvcfcombine%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfcombine/vcfcombine/0.0.2",
    "name": "VCFcombine:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Break multiple alleles into multiple records, or combine overallpoing alleles into a single record",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfbreakcreatemulti%2Fvcfbreakcreatemulti%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfbreakcreatemulti/vcfbreakcreatemulti/0.0.2",
    "name": "VCFbreakCreateMulti:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Split alleleic primitives (gaps or mismatches) into multiple VCF lines",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfallelicprimitives%2Fvcfallelicprimitives%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfallelicprimitives/vcfallelicprimitives/0.0.2",
    "name": "VcfAllelicPrimitives:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "filter VCF data in a variety of attributes",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcffilter%2Fvcffilter%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcffilter/vcffilter/0.0.2",
    "name": "VCFfilter:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Randomly sample sites from VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfrandomsample%2Fvcfrandomsample%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfrandomsample/vcfrandomsample/0.0.2",
    "name": "VCFrandomSample:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Convert genotype-based phased alleles into haplotype alleles",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfgeno2haplo%2Fvcfgeno2haplo%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfgeno2haplo/vcfgeno2haplo/0.0.2",
    "name": "VCFgenotype-to-haplotype:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Output records belonging to samples commong between two datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfcommonsamples%2Fvcfcommonsamples%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfcommonsamples/vcfcommonsamples/0.0.2",
    "name": "VCFcommonSamples:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Select samples from a VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfselectsamples%2Fvcfselectsamples%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfselectsamples/vcfselectsamples/0.0.2",
    "name": "VCFselectsamples:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Left-align indels and complex variants in VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfleftalign%2Fvcfleftalign%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfleftalign/vcfleftalign/0.0.2",
    "name": "VCFleftAlign:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Convert VCF data into TAB-delimited format",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcf2tsv%2Fvcf2tsv%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcf2tsv/vcf2tsv/0.0.2",
    "name": "VCFtoTab-delimited:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Adds info fields from the second dataset which are not present in the first dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfaddinfo%2Fvcfaddinfo%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfaddinfo/vcfaddinfo/0.0.2",
    "name": "VCFaddinfo:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Intersect VCF records with BED annotations",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfannotate%2Fvcfannotate%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfannotate/vcfannotate/0.0.2",
    "name": "VCFannotate:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Annotate genotypes in a VCF dataset using genotypes from another VCF dataset",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfannotategenotypes%2Fvcfannotategenotypes%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfannotategenotypes/vcfannotategenotypes/0.0.2",
    "name": "VCFannotateGenotypes:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Verify that the reference allele matches the reference genome",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfcheck%2Fvcfcheck%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfcheck/vcfcheck/0.0.2",
    "name": "VCFcheck:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Sort VCF dataset by coordinate",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfsort%2Fvcfsort%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfsort/vcfsort/0.0.1",
    "name": "VCFsort:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Convert numerical representation of genotypes to allelic",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfgenotypes%2Fvcfgenotypes%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfgenotypes/vcfgenotypes/0.0.2",
    "name": "VCFgenotypes:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Count the number of heterozygotes and alleles, compute het/hom ratio",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfhethom%2Fvcfhethom%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfhethom/vcfhethom/0.0.2",
    "name": "VCFhetHomAlleles:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Intersect VCF and BED datasets",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfbedintersect%2Fvcfbedintersect%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfbedintersect/vcfbedintersect/0.0.2",
    "name": "VCF-BEDintersect:"
  }, {
    "panel_section_name": "NGS: VCF Manipulation",
    "description": "Calculate distance to the nearest variant",
    "panel_section_id": "ngs:_vcf_manipulation",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fanton%2Fvcfdistance%2Fvcfdistance%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/anton/vcfdistance/vcfdistance/0.0.2",
    "name": "VCFdistance:"
  }], "model_class": "ToolSection", "version": "", "id": "ngs:_vcf_manipulation", "name": "NGS: VCF Manipulation"
}, {
  "elems": [{
    "panel_section_name": "Extract Features",
    "description": "expander",
    "panel_section_id": "features",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=gene2exon1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "gene2exon1",
    "name": "Gene BED To Exon/Intron/Codon BED"
  }], "model_class": "ToolSection", "version": "", "id": "features", "name": "Extract Features"
}, {
  "elems": [{
    "panel_section_name": "Fetch Sequences",
    "description": "using coordinates from assembled/unassembled genomes",
    "panel_section_id": "fetchSeq",
    "version": "2.2.3",
    "link": "/tool_runner?tool_id=Extract+genomic+DNA+1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Extract genomic DNA 1",
    "name": "Extract Genomic DNA"
  }], "model_class": "ToolSection", "version": "", "id": "fetchSeq", "name": "Fetch Sequences"
}, {
  "elems": [{
    "panel_section_name": "Fetch Alignments",
    "description": "given a set of genomic intervals",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=Interval2Maf_pairwise1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Interval2Maf_pairwise1",
    "name": "Extract Pairwise MAF blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "given a set of genomic intervals",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=Interval2Maf1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Interval2Maf1",
    "name": "Extract MAF blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "given a set of genomic intervals",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=Interval_Maf_Merged_Fasta2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Interval_Maf_Merged_Fasta2",
    "name": "Stitch MAF blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "given a set of coding exon intervals",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=GeneBed_Maf_Fasta2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "GeneBed_Maf_Fasta2",
    "name": "Stitch Gene blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "Alignment coverage information",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=maf_stats1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "maf_stats1",
    "name": "MAF Coverage Stats"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "by Species",
    "panel_section_id": "fetchAlign",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=MAF_Thread_For_Species1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_Thread_For_Species1",
    "name": "Join MAF blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "by Species",
    "panel_section_id": "fetchAlign",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=MAF_Limit_To_Species1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_Limit_To_Species1",
    "name": "Filter MAF blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "by Size",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=maf_limit_size1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "maf_limit_size1",
    "name": "Filter MAF blocks"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "given a set of block numbers and a MAF file",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=maf_by_block_number1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "maf_by_block_number1",
    "name": "Extract MAF by block number"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "by specified attributes",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=MAF_filter",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_filter",
    "name": "Filter MAF"
  }, {
    "panel_section_name": "Fetch Alignments",
    "description": "a MAF file",
    "panel_section_id": "fetchAlign",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=MAF_Reverse_Complement_1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "MAF_Reverse_Complement_1",
    "name": "Reverse Complement"
  }], "model_class": "ToolSection", "version": "", "id": "fetchAlign", "name": "Fetch Alignments"
}, {
  "elems": [{
    "panel_section_name": "Get Genomic Scores",
    "description": "converter",
    "panel_section_id": "scores",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=wiggle2simple1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "wiggle2simple1",
    "name": "Wiggle-to-Interval"
  }, {
    "panel_section_name": "Get Genomic Scores",
    "description": "such as phastCons, GERP, binCons, and others for a set of genomic intervals",
    "panel_section_id": "scores",
    "version": "1.1.3",
    "link": "/tool_runner?tool_id=aggregate_scores_in_intervals2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "aggregate_scores_in_intervals2",
    "name": "Aggregate datapoints"
  }], "model_class": "ToolSection", "version": "", "id": "scores", "name": "Get Genomic Scores"
}, {
  "elems": [{
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "for a set of genomic intervals",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fannotation_profiler%2FAnnotation_Profiler_0%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/annotation_profiler/Annotation_Profiler_0/1.0.0",
    "name": "Profile Annotations"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "the overlapping intervals of a dataset",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmerge%2Fgops_merge_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/merge/gops_merge_1/1.0.0",
    "name": "Merge"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "for every interval",
    "panel_section_id": "bxops",
    "version": "4.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fflanking_features%2Fflanking_features_1%2F4.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/flanking_features/flanking_features_1/4.0.1",
    "name": "Fetch closest non-overlapping feature"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "two datasets into one dataset",
    "panel_section_id": "bxops",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fconcat%2Fgops_concat_1%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/concat/gops_concat_1/1.0.1",
    "name": "Concatenate"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "the intervals of two datasets",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsubtract%2Fgops_subtract_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/subtract/gops_subtract_1/1.0.0",
    "name": "Subtract"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "the intervals of two datasets side-by-side",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fjoin%2Fgops_join_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/join/gops_join_1/1.0.0",
    "name": "Join"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "the intervals of two datasets",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fintersect%2Fgops_intersect_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/intersect/gops_intersect_1/1.0.0",
    "name": "Intersect"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "returns flanking region/s for every gene",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fget_flanks%2Fget_flanks1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/get_flanks/get_flanks1/1.0.0",
    "name": "Get flanks"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "of a set of intervals on second set of intervals",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcoverage%2Fgops_coverage_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/coverage/gops_coverage_1/1.0.0",
    "name": "Coverage"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "intervals of a dataset",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcomplement%2Fgops_complement_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/complement/gops_complement_1/1.0.0",
    "name": "Complement"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "the intervals of a dataset",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcluster%2Fgops_cluster_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/cluster/gops_cluster_1/1.0.0",
    "name": "Cluster"
  }, {
    "panel_section_name": "Operate on Genomic Intervals",
    "description": "of all intervals",
    "panel_section_id": "bxops",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbasecoverage%2Fgops_basecoverage_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/basecoverage/gops_basecoverage_1/1.0.0",
    "name": "Base Coverage"
  }], "model_class": "ToolSection", "version": "", "id": "bxops", "name": "Operate on Genomic Intervals"
}, {
  "elems": [{
    "panel_section_name": "Statistics",
    "description": "on \"Perform LDA\" output",
    "panel_section_id": "stats",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fplot_from_lda%2Fplot_for_lda_output1%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/plot_from_lda/plot_for_lda_output1/1.0.1",
    "name": "Draw ROC plot"
  }, {
    "panel_section_name": "Statistics",
    "description": "Linear Discriminant Analysis",
    "panel_section_id": "stats",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flda_analysis%2Flda_analy1%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/lda_analysis/lda_analy1/1.0.1",
    "name": "Perform LDA"
  }, {
    "panel_section_name": "Statistics",
    "description": "for using PC and LDA",
    "panel_section_id": "stats",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgenerate_pc_lda_matrix%2Fgenerate_matrix_for_pca_and_lda1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/generate_pc_lda_matrix/generate_matrix_for_pca_and_lda1/1.0.0",
    "name": "Generate A Matrix"
  }, {
    "panel_section_name": "Statistics",
    "description": "for numeric columns",
    "panel_section_id": "stats",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcorrelation%2Fcor2%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/correlation/cor2/1.0.0",
    "name": "Correlation"
  }, {
    "panel_section_name": "Statistics",
    "description": "for any numerical column",
    "panel_section_id": "stats",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=Summary_Statistics1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Summary_Statistics1",
    "name": "Summary Statistics"
  }, {
    "panel_section_name": "Statistics",
    "description": "occurrences of each record",
    "panel_section_id": "stats",
    "version": "1.0.2",
    "link": "/tool_runner?tool_id=Count1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "Count1",
    "name": "Count"
  }], "model_class": "ToolSection", "version": "", "id": "stats", "name": "Statistics"
}, {
  "elems": [{
    "panel_section_name": "Graph/Display Data",
    "description": "for UCSC genome browser",
    "panel_section_id": "plots",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fucsc_custom_track%2Fbuild_ucsc_custom_track_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/ucsc_custom_track/build_ucsc_custom_track_1/1.0.0",
    "name": "Build custom track"
  }, {
    "panel_section_name": "Graph/Display Data",
    "description": "of two numeric columns",
    "panel_section_id": "plots",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fscatterplot%2Fscatterplot_rpy%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/scatterplot/scatterplot_rpy/1.0.0",
    "name": "Scatterplot"
  }, {
    "panel_section_name": "Graph/Display Data",
    "description": "of a numeric column",
    "panel_section_id": "plots",
    "version": "1.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fhistogram%2Fhistogram_rpy%2F1.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.3",
    "name": "Histogram"
  }, {
    "panel_section_name": "Graph/Display Data",
    "description": "for multiple series and graph types",
    "panel_section_id": "plots",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fxy_plot%2FXY_Plot_1%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/xy_plot/XY_Plot_1/1.0.1",
    "name": "Plotting tool"
  }, {
    "panel_section_name": "Graph/Display Data",
    "description": "Multiple Alignment Viewer",
    "panel_section_id": "plots",
    "version": "2.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgmaj%2Fgmaj_1%2F2.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/gmaj/gmaj_1/2.0.1",
    "name": "GMAJ"
  }, {
    "panel_section_name": "Graph/Display Data",
    "description": "of quality statistics",
    "panel_section_id": "plots",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=qual_stats_boxplot",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "qual_stats_boxplot",
    "name": "Boxplot"
  }, {
    "panel_section_name": "Graph/Display Data",
    "description": "for display at UCSC",
    "panel_section_id": "plots",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=vcf_to_maf_customtrack1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "vcf_to_maf_customtrack1",
    "name": "VCF to MAF Custom Track"
  }], "model_class": "ToolSection", "version": "", "id": "plots", "name": "Graph/Display Data"
}, {
  "elems": [{
    "panel_section_name": "Phenotype Association",
    "description": "Convert from VCF to pgSnp format",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcf2pgsnp%2Fvcf2pgSnp%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcf2pgsnp/vcf2pgSnp/1.0.0",
    "name": "VCF to pgSnp"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "significant SNPs in case-control data",
    "panel_section_id": "hgv",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsnpfreq%2Fhgv_snpFreq%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/snpfreq/hgv_snpFreq/1.0.1",
    "name": "snpFreq"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "analysis of chemicals, diseases, or genes",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fctd_batch%2FctdBatch_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/ctd_batch/ctdBatch_1/1.0.0",
    "name": "CTD"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "into columns",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdivide_pg_snp%2FdividePgSnp%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/divide_pg_snp/dividePgSnp/1.0.0",
    "name": "Separate pgSnp alleles"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "human genes associated with disease terms",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fhgv_fundo%2Fhgv_funDo%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/hgv_fundo/hgv_funDo/1.0.0",
    "name": "FunDO"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "visualization of genomic data with the Hilbert curve",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fhgv_hilbertvis%2Fhgv_hilbertvis%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/hgv_hilbertvis/hgv_hilbertvis/1.0.0",
    "name": "HVIS"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "Bayesian genetic variant detector",
    "panel_section_id": "hgv",
    "version": "0.5.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffreebayes_wrapper%2Ffreebayes_wrapper%2F0.5.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/freebayes_wrapper/freebayes_wrapper/0.5.0",
    "name": "Call SNPS with Freebayes"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "database info",
    "panel_section_id": "hgv",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdgidb_annotator%2Fdgidb_annotator%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/dgidb_annotator/dgidb_annotator/0.1",
    "name": "Annotate with DGI"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "amino-acid changes caused by a set of SNPs",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_codingSnps",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_codingSnps",
    "name": "aaChanges"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "interspecies conservation scores",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_add_scores",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_add_scores",
    "name": "phyloP"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "predictions of functional sites",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_sift",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_sift",
    "name": "SIFT"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "tools for functional profiling of gene lists",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_linkToGProfile",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_linkToGProfile",
    "name": "g:Profiler"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "functional annotation for a list of genes",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_david",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_david",
    "name": "DAVID"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "linkage disequilibrium and tag SNPs",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_ldtools",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_ldtools",
    "name": "LD"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "significant transcription factor binding sites from ChIP data",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_pass",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_pass",
    "name": "PASS"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "significant single-SNP associations in case-control studies",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_gpass",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_gpass",
    "name": "GPASS"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "significant single- and multi-locus SNP associations in case-control studies",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_beam",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_beam",
    "name": "BEAM"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "LASSO-Patternsearch algorithm",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=hgv_lps",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "hgv_lps",
    "name": "LPS"
  }, {
    "panel_section_name": "Phenotype Association",
    "description": "Convert from MasterVar to pgSnp format",
    "panel_section_id": "hgv",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=master2pgSnp",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "master2pgSnp",
    "name": "MasterVar to pgSnp"
  }], "model_class": "ToolSection", "version": "", "id": "hgv", "name": "Phenotype Association"
}, {
  "elems": [{
    "panel_section_name": "snpEff",
    "description": "Filter variants using arbitrary expressions",
    "panel_section_id": "snpeff",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpcingola%2Fsnpeff%2FsnpSift_filter%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/pcingola/snpeff/snpSift_filter/1.0",
    "name": "SnpSift Filter"
  }, {
    "panel_section_name": "snpEff",
    "description": "Variant effect and annotation",
    "panel_section_id": "snpeff",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpcingola%2Fsnpeff%2FsnpEff%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/pcingola/snpeff/snpEff/1.0",
    "name": "SnpEff"
  }, {
    "panel_section_name": "snpEff",
    "description": "Marks up a snpEff output file with matches to a gene candidate list.",
    "panel_section_id": "snpeff",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgregory-minevich%2Fcheck_snpeff_candidates%2Fcheck_snpeff_candidates%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/gregory-minevich/check_snpeff_candidates/check_snpeff_candidates/1.0.0",
    "name": "CloudMap: Check snpEff Candidates"
  }], "model_class": "ToolSection", "version": "", "id": "snpeff", "name": "snpEff"
}, {
  "elems": [{
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_intersectbed_bam%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_intersectbed_bam/0.1.0",
    "name": "Intersect BAM alignments with intervals in another files"
  }, {
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_coveragebed_counts%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_coveragebed_counts/0.1.0",
    "name": "Count intervals in one file overlapping intervals in another file"
  }, {
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_genomecoveragebed_histogram%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_genomecoveragebed_histogram/0.1.0",
    "name": "Create a histogram of genome coverage"
  }, {
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_genomecoveragebed_bedgraph%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_genomecoveragebed_bedgraph/0.1.0",
    "name": "Create a BedGraph of genome coverage"
  }, {
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_bamtobed%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_bamtobed/0.1.0",
    "name": "Convert from BAM to BED"
  }, {
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_mergebedgraph%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_mergebedgraph/0.1.0",
    "name": "Merge BedGraph files"
  }, {
    "panel_section_name": "BEDTools",
    "description": "",
    "panel_section_id": "bedtools",
    "version": "0.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Faaronquinlan%2Fbedtools%2Fbedtools_multiintersectbed%2F0.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_multiintersectbed/0.1.0",
    "name": "Intersect multiple sorted BED files"
  }], "model_class": "ToolSection", "version": "", "id": "bedtools", "name": "BEDTools"
}, {
  "elems": [{
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gd_init",
    "text": "Basic analyses"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Examine sequence coverage for SNPs",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_coverage_distributions%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_coverage_distributions/1.0.0",
    "name": "Coverage Distributions"
  }, {
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gd_pop",
    "text": "Population structure"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Principal Components Analysis of genotype data",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_pca%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_pca/1.0.0",
    "name": "PCA"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Evaluate possible numbers of ancestral populations",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_evaluate_population_numbers%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_evaluate_population_numbers/1.0.0",
    "name": "Population Complexity"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Characterize ancestries w.r.t. inferred ancestral populations",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_population_structure%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_population_structure/1.0.0",
    "name": "Ancestry"
  }, {
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gd_kegg",
    "text": "KEGG and GO"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Look up KEGG pathways for given Ensembl transcripts",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_new_oscar%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_new_oscar/1.0.0",
    "name": "Get Pathways"
  }, {
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gd_geno",
    "text": "Design genotyping studies"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Select a specified number of SNPs, uniformly spaced",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_select_snps%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_select_snps/1.0.0",
    "name": "Sample SNPs"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Select SNPs differentially cut by specified restriction enzymes",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_specify_restriction_enzymes%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_specify_restriction_enzymes/1.0.0",
    "name": "Differential Cleavage"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Fetch DNA sequence for intervals surrounding the given SNPs",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_extract_flanking_dna%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_extract_flanking_dna/1.0.0",
    "name": "Flanking Sequence"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Find suitable PCR primers for SNPs",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_extract_primers%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_extract_primers/1.0.0",
    "name": "Pick Primers"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": &pi; and &theta;",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_nucleotide_diversity_pi%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_nucleotide_diversity_pi/1.0.0",
    "name": "Nucleotide Diversity"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Find high-scoring runs of SNPs",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_find_intervals%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_find_intervals/1.1.0",
    "name": "Remarkable Intervals"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Define a collection of individuals from a gd_snp dataset",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_specify%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_specify/1.1.0",
    "name": "Specify Individuals"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Show genetic relationships among individuals",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_phylogenetic_tree%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_phylogenetic_tree/1.1.0",
    "name": "Phylogenetic Tree"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Estimate the relative fixation index between two populations",
    "panel_section_id": "genome_diversity",
    "version": "1.3.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_average_fst%2F1.3.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_average_fst/1.3.0",
    "name": "Overall FST"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Compute a fixation index score for each SNP",
    "panel_section_id": "genome_diversity",
    "version": "1.2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_add_fst_column%2F1.2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_add_fst_column/1.2.0",
    "name": "Per-SNP FSTs"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Draw a KEGG pathway, highlighting specified gene modules",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_pathway_image%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_pathway_image/1.1.0",
    "name": "Pathway Image"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Build a gd_snp or gd_genotype file",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_make_gd_file%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_make_gd_file/1.0.0",
    "name": "Make File"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": CSV, FSTAT, Genepop or VCF to either gd_snp or gd_genotype",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_multiple_to_gd_genotype%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_multiple_to_gd_genotype/1.0.0",
    "name": "Convert"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Discard some SNPs based on coverage, quality or spacing",
    "panel_section_id": "genome_diversity",
    "version": "1.2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_filter_gd_snp%2F1.2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_filter_gd_snp/1.2.0",
    "name": "Filter SNPs"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Append summary columns for a population",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_sum_gd_snp%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_sum_gd_snp/1.1.0",
    "name": "Aggregate Individuals"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Fill in missing properties for a gd_snp or gd_genotype dataset",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_restore_attributes%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_restore_attributes/1.1.0",
    "name": "Restore Attributes"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Filter and convert to the format needed for these tools",
    "panel_section_id": "genome_diversity",
    "version": "1.2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_prepare_population_structure%2F1.2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_prepare_population_structure/1.2.0",
    "name": "Prepare Input"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Assess the impact of a gene set on KEGG pathways",
    "panel_section_id": "genome_diversity",
    "version": "1.2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_calc_freq%2F1.2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_calc_freq/1.2.0",
    "name": "Rank Pathways"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Assess the enrichment/depletion of a gene set for GO terms",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_rank_terms%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_rank_terms/1.1.0",
    "name": "Rank Terms"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Convert from gd_snp or gd_genotype to VCF format, for submission to dbSNP",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_gd_snp2vcf%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_gd_snp2vcf/1.1.0",
    "name": "gd_snp to VCF"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Map genomic intervals resembling specified source populations",
    "panel_section_id": "genome_diversity",
    "version": "1.2.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_dpmix%2F1.2.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_dpmix/1.2.0",
    "name": "Admixture"
  }, {
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gd_kin",
    "text": "Kinship"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Discover familial relationships",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_discover_familial_relationships%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_discover_familial_relationships/1.0.0",
    "name": "Close relatives"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Offspring estimated heterozygosity of sequenced pairs",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_offspring_heterozygosity%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_offspring_heterozygosity/1.0.0",
    "name": "Pairs sequenced"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Offspring estimated heterozygosity from a pedigree with sequenced founders",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_offspring_heterozygosity_pedigree%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_offspring_heterozygosity_pedigree/1.0.0",
    "name": "Founders sequenced"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Assignment of optimal breeding pairs",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_assignment_of_optimal_breeding_pairs%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_assignment_of_optimal_breeding_pairs/1.0.0",
    "name": "Matings"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Analyze the pedigree without genomic data",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_inbreeding_and_kinship%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_inbreeding_and_kinship/1.0.0",
    "name": "Inbreeding and kinship"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": Group gene categories connected by shared genes",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_cluster_kegg%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_cluster_kegg/1.0.0",
    "name": "Cluster KEGG"
  }, {
    "model_class": "ToolSectionLabel",
    "version": "",
    "id": "gd_mito",
    "text": "Mitochondria"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": pi, allowing for unsequenced intervals",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_diversity_pi%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_diversity_pi/1.1.0",
    "name": "Diversity"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": show positions of SNVs and unsequenced intervals",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_draw_variants%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_draw_variants/1.1.0",
    "name": "Draw variants"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": exchange rows in the above picture",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_reorder%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_reorder/1.0.0",
    "name": "Reorder individuals"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": construct a maximum-likelihood phylogenetic tree",
    "panel_section_id": "genome_diversity",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_raxml%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_raxml/1.0.0",
    "name": "RAxML"
  }, {
    "panel_section_name": "Genome Diversity",
    "description": ": prepare data for phylogenetic analysis",
    "panel_section_id": "genome_diversity",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmiller-lab%2Fgenome_diversity%2Fgd_make_phylip%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/miller-lab/genome_diversity/gd_make_phylip/1.1.0",
    "name": "Phylip"
  }], "model_class": "ToolSection", "version": "", "id": "genome_diversity", "name": "Genome Diversity"
}, {
  "elems": [{
    "panel_section_name": "EMBOSS",
    "description": "Displays protein hydropathy",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepwindow73%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepwindow73/5.0.0",
    "name": "pepwindow"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays protein hydropathy of a set of sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepwindowall74%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepwindowall74/5.0.0",
    "name": "pepwindowall"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Removes gap characters from sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+degapseq20%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: degapseq20/5.0.0",
    "name": "degapseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays some simple information about sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+infoseq46%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: infoseq46/5.0.0",
    "name": "infoseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Needleman-Wunsch global alignment",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+needle56%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: needle56/5.0.0",
    "name": "needle"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Search a protein sequence with a motif",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+patmatdb67%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: patmatdb67/5.0.0",
    "name": "patmatdb"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds MAR/SAR sites in nucleic sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+marscan49%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: marscan49/5.0.0",
    "name": "marscan"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Find protein sequence regions with a biased composition",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+oddcomp64%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: oddcomp64/5.0.0",
    "name": "oddcomp"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Protein pattern search after translation",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+fuzztran39%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: fuzztran39/5.0.0",
    "name": "fuzztran"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Report nucleic acid binding motifs",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+helixturnhelix43%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: helixturnhelix43/5.0.0",
    "name": "helixturnhelix"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Merge two overlapping nucleic acid sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+merger54%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: merger54/5.0.0",
    "name": "merger"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reports CpG rich region",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+newcpgseek58%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: newcpgseek58/5.0.0",
    "name": "newcpgseek"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Extract regions from a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+extractseq35%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: extractseq35/5.0.0",
    "name": "extractseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Mask off features of a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+maskfeat50%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: maskfeat50/5.0.0",
    "name": "maskfeat"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Codon usage table comparison",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+codcmp12%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: codcmp12/5.0.0",
    "name": "codcmp"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Match large sequences against one or more other sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+supermatcher95%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: supermatcher95/5.0.0",
    "name": "supermatcher"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds tandem repeats",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+equicktandem31%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: equicktandem31/5.0.0",
    "name": "equicktandem"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Removes a specified section from a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cutseq18%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cutseq18/5.0.0",
    "name": "cutseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Align nucleic coding regions given the aligned proteins",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+tranalign100%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: tranalign100/5.0.0",
    "name": "tranalign"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "CAI codon adaptation index using custom codon usage file",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cai_custom6%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cai_custom6/5.0.0",
    "name": "cai custom"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Report CpG rich areas",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+newcpgreport57%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: newcpgreport57/5.0.0",
    "name": "newcpgreport"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Removes carriage return from ASCII files",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+noreturn60%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: noreturn60/5.0.0",
    "name": "noreturn"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reports protein signal cleavage sites",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+sigcleave88%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: sigcleave88/5.0.0",
    "name": "sigcleave"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reads sequence fragments and builds one sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+union105%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: union105/5.0.0",
    "name": "union"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Regular expression search of a protein sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+preg78%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: preg78/5.0.0",
    "name": "preg"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Mutate sequence beyond all recognition",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+msbar55%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: msbar55/5.0.0",
    "name": "msbar"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds and extracts open reading frames (ORFs)",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+getorf42%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: getorf42/5.0.0",
    "name": "getorf"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Protein statistics",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepstats71%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepstats71/5.0.0",
    "name": "pepstats"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Protein proteolytic enzyme or reagent cleavage digest",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+digest23%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: digest23/5.0.0",
    "name": "digest"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Extract features from a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+extractfeat34%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: extractfeat34/5.0.0",
    "name": "extractfeat"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reverse and complement a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+revseq82%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: revseq82/5.0.0",
    "name": "revseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reads and writes sequences, skipping first few",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+skipseq91%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: skipseq91/5.0.0",
    "name": "skipseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Split a sequence into (overlapping) smaller sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+splitter92%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: splitter92/5.0.0",
    "name": "splitter"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Calculates the isoelectric point of a protein",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+iep45%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: iep45/5.0.0",
    "name": "iep"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Translate nucleic acid sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+transeq101%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: transeq101/5.0.0",
    "name": "transeq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Fickett TESTCODE statistic to identify protein-coding DNA",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+tcode97%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: tcode97/5.0.0",
    "name": "tcode"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays protein hydropathy",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+octanol63%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: octanol63/5.0.0",
    "name": "octanol"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Bending and curvature plot in B-DNA",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+banana3%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: banana3/5.0.0",
    "name": "banana"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Mask off regions of a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+maskseq51%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: maskseq51/5.0.0",
    "name": "maskseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Calculates fractional GC content of nucleic acid sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+geecee41%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: geecee41/5.0.0",
    "name": "geecee"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Shows protein sequences as helices",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepwheel72%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepwheel72/5.0.0",
    "name": "pepwheel"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Plot CpG rich areas",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cpgplot15%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cpgplot15/5.0.0",
    "name": "cpgplot"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Exclude a set of sequences and write out the remaining ones",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+notseq61%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: notseq61/5.0.0",
    "name": "notseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Smith-Waterman local alignment",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+water107%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: water107/5.0.0",
    "name": "water"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reads and writes sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+seqret84%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: seqret84/5.0.0",
    "name": "seqret"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Plots simple amino acid properties in parallel",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepinfo69%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepinfo69/5.0.0",
    "name": "pepinfo"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Merge two large overlapping nucleic acid sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+megamerger53%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: megamerger53/5.0.0",
    "name": "megamerger"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Trim ambiguous bits off the ends of sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+trimseq103%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: trimseq103/5.0.0",
    "name": "trimseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Draws linear maps of DNA constructs",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+lindna48%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: lindna48/5.0.0",
    "name": "lindna"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Looks for inverted repeats in a nucleotide sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+palindrome65%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: palindrome65/5.0.0",
    "name": "palindrome"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Insert one sequence into another",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pasteseq66%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pasteseq66/5.0.0",
    "name": "pasteseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Show features of a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+showfeat85%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: showfeat85/5.0.0",
    "name": "showfeat"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Plot potential open reading frames",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+plotorf76%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: plotorf76/5.0.0",
    "name": "plotorf"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Looks for tandem repeats in a nucleotide sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+etandem33%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: etandem33/5.0.0",
    "name": "etandem"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "CAI codon adaptation index",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cai6%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cai6/5.0.0",
    "name": "cai"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Regular expression search of a nucleotide sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dreg27%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: dreg27/5.0.0",
    "name": "dreg"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Synonymous codon usage Gribskov statistic plot",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+syco96%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: syco96/5.0.0",
    "name": "syco"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Count composition of dimer/trimer/etc words in a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+compseq14%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: compseq14/5.0.0",
    "name": "compseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Display a DNA sequence with 6-frame translation and ORFs",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+sixpack90%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: sixpack90/5.0.0",
    "name": "sixpack"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Calculates the twisting in a B-DNA sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+btwisted5%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: btwisted5/5.0.0",
    "name": "btwisted"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Output sequence with translated ranges",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+prettyseq80%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: prettyseq80/5.0.0",
    "name": "prettyseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Find differences between nearly identical sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+diffseq22%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: diffseq22/5.0.0",
    "name": "diffseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Type in a short new sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+newseq59%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: newseq59/5.0.0",
    "name": "newseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays proteins as a helical net",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepnet70%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepnet70/5.0.0",
    "name": "pepnet"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Plot quality of conservation of a sequence alignment",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+plotcon75%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: plotcon75/5.0.0",
    "name": "plotcon"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays a wordmatch dotplot of two sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dottup26%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: dottup26/5.0.0",
    "name": "dottup"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Create a chaos game representation plot for a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+chaos7%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: chaos7/5.0.0",
    "name": "chaos"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Search sequence documentation. Slow, use SRS and Entrez!",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+textsearch98%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: textsearch98/5.0.0",
    "name": "textsearch"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Hydrophobic moment calculation",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+hmoment44%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: hmoment44/5.0.0",
    "name": "hmoment"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays aligned sequences, with colouring and boxing",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+prettyplot79%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: prettyplot79/5.0.0",
    "name": "prettyplot"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Protein pattern search",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+fuzzpro38%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: fuzzpro38/5.0.0",
    "name": "fuzzpro"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Searches DNA sequences for matches with primer pairs",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+primersearch81%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: primersearch81/5.0.0",
    "name": "primersearch"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Wobble base plot",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+wobble108%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: wobble108/5.0.0",
    "name": "wobble"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Strips out DNA between a pair of vector sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+vectorstrip106%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: vectorstrip106/5.0.0",
    "name": "vectorstrip"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "All-against-all comparison of a set of sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+seqmatchall83%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: seqmatchall83/5.0.0",
    "name": "seqmatchall"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Counts words of a specified size in a DNA sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+wordcount109%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: wordcount109/5.0.0",
    "name": "wordcount"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Shuffles a set of sequences maintaining composition",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+shuffleseq87%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: shuffleseq87/5.0.0",
    "name": "shuffleseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Predicts protein secondary structure",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+garnier40%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: garnier40/5.0.0",
    "name": "garnier"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Trim poly-A tails off EST sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+trimest102%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: trimest102/5.0.0",
    "name": "trimest"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Create a codon usage table",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cusp17%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cusp17/5.0.0",
    "name": "cusp"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Non-overlapping wordmatch dotplot of two sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dotpath25%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: dotpath25/5.0.0",
    "name": "dotpath"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds the best local alignments between two sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+matcher52%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: matcher52/5.0.0",
    "name": "matcher"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Alter the name or description of a sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+descseq21%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: descseq21/5.0.0",
    "name": "descseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Codon usage statistics",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+chips10%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: chips10/5.0.0",
    "name": "chips"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds DNA inverted repeats",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+einverted28%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: einverted28/5.0.0",
    "name": "einverted"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Draws circular maps of DNA constructs",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cirdna11%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cirdna11/5.0.0",
    "name": "cirdna"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays membrane spanning regions",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+tmap99%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: tmap99/5.0.0",
    "name": "tmap"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Writes one sequence from a multiple set of sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+nthseq62%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: nthseq62/5.0.0",
    "name": "nthseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reports STOP codons and ORF statistics of a protein",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+checktrans9%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: checktrans9/5.0.0",
    "name": "checktrans"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Reports all CpG rich regions",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cpgreport16%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: cpgreport16/5.0.0",
    "name": "cpgreport"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays all-against-all dotplots of a set of sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+polydot77%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: polydot77/5.0.0",
    "name": "polydot"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Calculates DNA RNA/DNA melting temperature",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dan19%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: dan19/5.0.0",
    "name": "dan"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds siRNA duplexes in mRNA",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+sirna89%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: sirna89/5.0.0",
    "name": "sirna"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Predicts coiled coil regions",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepcoil68%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: pepcoil68/5.0.0",
    "name": "pepcoil"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Plots isochores in large DNA sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+isochore47%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: isochore47/5.0.0",
    "name": "isochore"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Align EST and genomic DNA sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+est2genome32%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: est2genome32/5.0.0",
    "name": "est2genome"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds PEST motifs as potential proteolytic cleavage sites",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+epestfind29%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: epestfind29/5.0.0",
    "name": "epestfind"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Extract CDS, mRNA and translations from feature tables",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+coderet13%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: coderet13/5.0.0",
    "name": "coderet"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Protein charge plot",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+charge8%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: charge8/5.0.0",
    "name": "charge"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Residue/base frequency table or plot",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+freak36%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: freak36/5.0.0",
    "name": "freak"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Replace or delete sequence sections",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+biosed4%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: biosed4/5.0.0",
    "name": "biosed"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds neighbouring pairs of features in sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+twofeat104%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: twofeat104/5.0.0",
    "name": "twofeat"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Finds all exact matches of a given size between 2 sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+wordmatch110%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: wordmatch110/5.0.0",
    "name": "wordmatch"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Back translate a protein sequence",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+backtranseq2%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: backtranseq2/5.0.0",
    "name": "backtranseq"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Predicts potentially antigenic regions of a protein sequence, using the method of Kolaskar and Tongaonkar.",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+antigenic1%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: antigenic1/5.0.0",
    "name": "antigenic"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Displays a thresholded dotplot of two sequences",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dotmatcher24%2F5.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: dotmatcher24/5.0.0",
    "name": "dotmatcher"
  }, {
    "panel_section_name": "EMBOSS",
    "description": "Nucleic acid pattern search",
    "panel_section_id": "EMBOSSLite",
    "version": "5.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+fuzznuc37%2F5.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: fuzznuc37/5.0.1",
    "name": "fuzznuc"
  }], "model_class": "ToolSection", "version": "", "id": "EMBOSSLite", "name": "EMBOSS"
}, {
  "elems": [{
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flinear_regression%2FLinearRegression1%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/linear_regression/LinearRegression1/1.0.1",
    "name": "Perform Linear Regression"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flogistic_regression_vif%2FLogisticRegression%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/logistic_regression_vif/LogisticRegression/1.0.1",
    "name": "Perform Logistic Regression with vif"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpartialr_square%2FpartialRsq%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/partialr_square/partialRsq/1.0.0",
    "name": "Compute partial R square"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Frcve%2Frcve1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/rcve/rcve1/1.0.0",
    "name": "Compute RCVE"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "based on quality scores",
    "panel_section_id": "regVar",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fquality_filter%2FqualityFilter%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/quality_filter/qualityFilter/1.0.1",
    "name": "Filter nucleotides"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "for non-coding regions",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsubstitution_rates%2FsubRate1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/substitution_rates/subRate1/1.0.0",
    "name": "Estimate substitution rates "
  }, {
    "panel_section_name": "Regional Variation",
    "description": "from pairwise alignments",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsubstitutions%2Fsubstitutions1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/substitutions/substitutions1/1.0.0",
    "name": "Fetch substitutions "
  }, {
    "panel_section_name": "Regional Variation",
    "description": "of the values of features overlapping an interval",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fweightedaverage%2Fwtavg%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/weightedaverage/wtavg/1.0.0",
    "name": "Assign weighted-average"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcanonical_correlation_analysis%2Fcca1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/canonical_correlation_analysis/cca1/1.0.0",
    "name": "Canonical Correlation Analysis"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkernel_canonical_correlation_analysis%2Fkcca1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/kernel_canonical_correlation_analysis/kcca1/1.0.0",
    "name": "Kernel Canonical Correlation Analysis"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkernel_principal_component_analysis%2Fkpca1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/kernel_principal_component_analysis/kpca1/1.0.0",
    "name": "Kernel Principal Component Analysis"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fprincipal_component_analysis%2Fpca1%2F1.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/principal_component_analysis/pca1/1.0.2",
    "name": "Principal Component Analysis"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "from 3-way alignments",
    "panel_section_id": "regVar",
    "version": "1.0.3",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Findels_3way%2Findels_3way%2F1.0.3",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/indels_3way/indels_3way/1.0.3",
    "name": "Fetch Indels"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "2.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffeaturecounter%2FfeatureCoverage1%2F2.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/featurecounter/featureCoverage1/2.0.0",
    "name": "Feature coverage"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "for 3-way alignments",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgetindelrates_3way%2FindelRates_3way%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/getindelrates_3way/indelRates_3way/1.0.0",
    "name": "Estimate Indel Rates"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "from MAF file",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmaf_cpg_filter%2FcpgFilter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/maf_cpg_filter/cpgFilter/1.0.0",
    "name": "Mask CpG/non-CpG sites"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "by specified attributes",
    "panel_section_id": "regVar",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmicrosats_mutability%2Fmicrosats_mutability1%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/microsats_mutability/microsats_mutability1/1.1.0",
    "name": "Estimate microsatellite mutability"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbest_regression_subsets%2FBestSubsetsRegression1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/best_regression_subsets/BestSubsetsRegression1/1.0.0",
    "name": "Perform Best-subsets Regression"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "from pairwise alignments",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgetindels_2way%2FgetIndels_2way%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/getindels_2way/getIndels_2way/1.0.0",
    "name": "Fetch Indels"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "from pair-wise alignments",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmicrosats_alignment_level%2Fmicrosats_align1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/microsats_alignment_level/microsats_align1/1.0.0",
    "name": "Extract Orthologous Microsatellites"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fwindowsplitter%2FwinSplitter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/windowsplitter/winSplitter/1.0.0",
    "name": "Make windows"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "for multiple series",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fguru-ananda%2Fkaryotype_plot%2Fkaryotype_Plot_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/guru-ananda/karyotype_plot/karyotype_Plot_1/1.0.0",
    "name": "Karyotype Plotting tool"
  }, {
    "panel_section_name": "Regional Variation",
    "description": "on numeric data",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fguru-ananda%2Frhmm%2Fhmm_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/guru-ananda/rhmm/hmm_1/1.0.0",
    "name": "Fit HMM "
  }, {
    "panel_section_name": "Regional Variation",
    "description": "of numeric data",
    "panel_section_id": "regVar",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fguru-ananda%2Fheatmap%2Fheatmap_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/guru-ananda/heatmap/heatmap_1/1.0.0",
    "name": "Heatmap "
  }], "model_class": "ToolSection", "version": "", "id": "regVar", "name": "Regional Variation"
}, {
  "elems": [{
    "panel_section_name": "FASTA manipulation",
    "description": "converts tabular file to FASTA format",
    "panel_section_id": "fasta_manipulation",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftabular_to_fasta%2Ftab2fasta%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/tabular_to_fasta/tab2fasta/1.1.0",
    "name": "Tabular-to-FASTA"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "converter",
    "panel_section_id": "fasta_manipulation",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_to_tabular%2Ffasta2tab%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_to_tabular/fasta2tab/1.1.0",
    "name": "FASTA-to-Tabular"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "",
    "panel_section_id": "fasta_manipulation",
    "version": "1.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_filter_by_length%2Ffasta_filter_by_length%2F1.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_filter_by_length/fasta_filter_by_length/1.1",
    "name": "Filter sequences by length"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "FASTA alignment by species",
    "panel_section_id": "fasta_manipulation",
    "version": "0.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_concatenate_by_species%2Ffasta_concatenate0%2F0.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_concatenate_by_species/fasta_concatenate0/0.0.0",
    "name": "Concatenate"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "",
    "panel_section_id": "fasta_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_compute_length%2Ffasta_compute_length%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_compute_length/fasta_compute_length/1.0.0",
    "name": "Compute sequence length"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "formatter",
    "panel_section_id": "fasta_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_formatter%2Fcshl_fasta_formatter%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_formatter/cshl_fasta_formatter/1.0.0",
    "name": "FASTA Width"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "converter",
    "panel_section_id": "fasta_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_nucleotide_changer%2Fcshl_fasta_nucleotides_changer%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fasta_nucleotide_changer/cshl_fasta_nucleotides_changer/1.0.0",
    "name": "RNA/DNA"
  }, {
    "panel_section_name": "FASTA manipulation",
    "description": "sequences",
    "panel_section_id": "fasta_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_collapser%2Fcshl_fastx_collapser%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/fastx_collapser/cshl_fastx_collapser/1.0.0",
    "name": "Collapse"
  }], "model_class": "ToolSection", "version": "", "id": "fasta_manipulation", "name": "FASTA manipulation"
}, {
  "elems": [{
    "panel_section_name": "Evolution",
    "description": "with SNPs",
    "panel_section_id": "hyphy",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmutate_snp_codon%2Fmutate_snp_codon_1%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/mutate_snp_codon/mutate_snp_codon_1/1.0.0",
    "name": "Mutate Codons"
  }], "model_class": "ToolSection", "version": "", "id": "hyphy", "name": "Evolution"
}, {
  "elems": [{
    "panel_section_name": "Multiple Alignments",
    "description": "multiple sequence alignment program for DNA or proteins",
    "panel_section_id": "clustal",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fclustalw%2Fclustalw%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/clustalw/clustalw/0.1",
    "name": "ClustalW"
  }], "model_class": "ToolSection", "version": "", "id": "clustal", "name": "Multiple Alignments"
}, {
  "elems": [{
    "panel_section_name": "Metagenomic analyses",
    "description": "",
    "panel_section_id": "tax_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffind_diag_hits%2Ffind_diag_hits%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/find_diag_hits/find_diag_hits/1.0.0",
    "name": "Find diagnostic hits"
  }, {
    "panel_section_name": "Metagenomic analyses",
    "description": "",
    "panel_section_id": "tax_manipulation",
    "version": "1.1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgi2taxonomy%2FFetch+Taxonomic+Ranks%2F1.1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/gi2taxonomy/Fetch Taxonomic Ranks/1.1.0",
    "name": "Fetch taxonomic representation"
  }, {
    "panel_section_name": "Metagenomic analyses",
    "description": "",
    "panel_section_id": "tax_manipulation",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flca_wrapper%2Flca1%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/lca_wrapper/lca1/1.0.1",
    "name": "Find lowest diagnostic rank"
  }, {
    "panel_section_name": "Metagenomic analyses",
    "description": "",
    "panel_section_id": "tax_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpoisson2test%2Fpoisson2test%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/poisson2test/poisson2test/1.0.0",
    "name": "Poisson two-sample test"
  }, {
    "panel_section_name": "Metagenomic analyses",
    "description": "",
    "panel_section_id": "tax_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ft2ps%2FDraw_phylogram%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/t2ps/Draw_phylogram/1.0.0",
    "name": "Draw phylogeny"
  }, {
    "panel_section_name": "Metagenomic analyses",
    "description": "",
    "panel_section_id": "tax_manipulation",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ft2t_report%2Ft2t_report%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/t2t_report/t2t_report/1.0.0",
    "name": "Summarize taxonomy"
  }], "model_class": "ToolSection", "version": "", "id": "tax_manipulation", "name": "Metagenomic analyses"
}, {
  "elems": [{
    "panel_section_name": "Motif Tools",
    "description": "generator for fasta (eg Clustal alignments)",
    "panel_section_id": "motifs",
    "version": "0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fweblogo3%2Frgweblogo3%2F0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/weblogo3/rgweblogo3/0.4",
    "name": "Sequence Logo"
  }], "model_class": "ToolSection", "version": "", "id": "motifs", "name": "Motif Tools"
}, {
  "model_class": "ToolSectionLabel",
  "version": "",
  "id": "ngs",
  "text": "NGS Toolbox Beta"
}, {
  "elems": [{
    "panel_section_name": "NGS: Peak Calling",
    "description": "Model-based Analysis of ChIP-Seq",
    "panel_section_id": "peak_calling",
    "version": "1.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmacs%2Fpeakcalling_macs%2F1.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/macs/peakcalling_macs/1.0.1",
    "name": "MACS"
  }, {
    "panel_section_name": "NGS: Peak Calling",
    "description": "Statistical approach for the Identification of ChIP-Enriched Regions",
    "panel_section_id": "peak_calling",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsicer%2Fpeakcalling_sicer%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/sicer/peakcalling_sicer/0.0.1",
    "name": "SICER"
  }], "model_class": "ToolSection", "version": "", "id": "peak_calling", "name": "NGS: Peak Calling"
}, {
  "elems": [{
    "panel_section_name": "NGS: Variant Analysis",
    "description": "- bayesian genetic variant detector",
    "panel_section_id": "variant_detection",
    "version": "0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffreebayes%2Ffreebayes%2F0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/freebayes/freebayes/0.4",
    "name": "FreeBayes"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "indels in BAM datasets",
    "panel_section_id": "variant_detection",
    "version": "0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffreebayes%2Fbamleftalign%2F0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/freebayes/bamleftalign/0.4",
    "name": "BamLeftAlign"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "- tabulate variable sites from BAM datasets",
    "panel_section_id": "variant_detection",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fblankenberg%2Fnaive_variant_caller%2Fnaive_variant_caller%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/blankenberg/naive_variant_caller/naive_variant_caller/0.0.1",
    "name": "Naive Variant Caller"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Map a mutation by linkage to regions of high mutation density using WGS data",
    "panel_section_id": "variant_detection",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgregory-minevich%2Fems_variant_density_mapping%2Fems_variant_density_mapping%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/gregory-minevich/ems_variant_density_mapping/ems_variant_density_mapping/1.0.0",
    "name": "CloudMap: EMS Variant Density Mapping"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Map a mutation by plotting recombination frequencies resulting from crossing to a highly polymorphic strain",
    "panel_section_id": "variant_detection",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgregory-minevich%2Fsnp_mapping_using_wgs%2Fsnp_mapping_using_wgs%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/gregory-minevich/snp_mapping_using_wgs/snp_mapping_using_wgs/1.0.0",
    "name": "CloudMap: Hawaiian Variant Mapping with WGS data"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Map a mutation using in silico bulk segregant linkage analysis using variants that are already present in the mutant strain of interest (rather than those introduced by a cross to a polymorphic strain).",
    "panel_section_id": "variant_detection",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgregory-minevich%2Fcloudmap_variant_discovery_mapping%2Fcloudmap_variant_discovery_mapping%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/gregory-minevich/cloudmap_variant_discovery_mapping/cloudmap_variant_discovery_mapping/1.0.0",
    "name": "CloudMap: Variant Discovery Mapping with WGS data"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Perform in silico complementation analysis on multiple tabular snpEff output files",
    "panel_section_id": "variant_detection",
    "version": "1.0.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgregory-minevich%2Fcloudmap_in_silico_complementation%2Fin_silico_complementation%2F1.0.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/gregory-minevich/cloudmap_in_silico_complementation/in_silico_complementation/1.0.0",
    "name": "CloudMap: in silico complementation"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "process variant counts",
    "panel_section_id": "variant_detection",
    "version": "1.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnick%2Fallele_counts%2Fallele_counts_1%2F1.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/nick/allele_counts/allele_counts_1/1.1",
    "name": "Variant Annotator"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Relatedness of minor alelle sequences in NJ tree",
    "panel_section_id": "variant_detection",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fboris%2Fphylorelatives%2Fphylorelatives%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/boris/phylorelatives/phylorelatives/0.0.1",
    "name": "Phylorelatives"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Minor Allele Frequency Boxplot",
    "panel_section_id": "variant_detection",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fboris%2Fhetbox%2Fhetbox%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/boris/hetbox/hetbox/1.0",
    "name": "MAF boxplot"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "Generate major and minor allele sequences from alleles table",
    "panel_section_id": "variant_detection",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fboris%2Fgetalleleseq%2Fgetalleleseq%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/boris/getalleleseq/getalleleseq/0.0.1",
    "name": "FASTA from allele counts"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "for variant detection",
    "panel_section_id": "variant_detection",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvarscan_version_2%2Fvarscan%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/varscan_version_2/varscan/0.1",
    "name": "Varscan"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "with functional information using ANNOVAR",
    "panel_section_id": "variant_detection",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftable_annovar%2Ftable_annovar%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/table_annovar/table_annovar/0.1",
    "name": "ANNOVAR Annotate VCF"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "a VCF dataset with custom filters",
    "panel_section_id": "variant_detection",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcftools_annotate%2Fvcftools_annotate%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcftools_annotate/vcftools_annotate/0.1",
    "name": "Annotate"
  }, {
    "panel_section_name": "NGS: Variant Analysis",
    "description": "to get data from selected regions",
    "panel_section_id": "variant_detection",
    "version": "0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcftools_slice%2Fvcftools_slice%2F0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/vcftools_slice/vcftools_slice/0.1",
    "name": "Slice VCF"
  }], "model_class": "ToolSection", "version": "", "id": "variant_detection", "name": "NGS: Variant Analysis"
}, {
  "elems": [{
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "",
    "panel_section_id": "gatk",
    "version": "0.0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariants_validate%2Fgatk_validate_variants%2F0.0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variants_validate/gatk_validate_variants/0.0.4",
    "name": "Validate Variants"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "from VCF files",
    "panel_section_id": "gatk",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_select%2Fgatk_variant_select%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_select/gatk_variant_select/0.0.2",
    "name": "Select Variants"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "",
    "panel_section_id": "gatk",
    "version": "0.0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_recalibrator%2Fgatk_variant_recalibrator%2F0.0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_recalibrator/gatk_variant_recalibrator/0.0.4",
    "name": "Variant Recalibrator"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "on VCF files",
    "panel_section_id": "gatk",
    "version": "0.0.5",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_filtration%2Fgatk_variant_filtration%2F0.0.5",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_filtration/gatk_variant_filtration/0.0.5",
    "name": "Variant Filtration"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "",
    "panel_section_id": "gatk",
    "version": "0.0.8",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_eval%2Fgatk_variant_eval%2F0.0.8",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_eval/gatk_variant_eval/0.0.8",
    "name": "Eval Variants"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "",
    "panel_section_id": "gatk",
    "version": "0.0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_combine%2Fgatk_variant_combine%2F0.0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_combine/gatk_variant_combine/0.0.4",
    "name": "Combine Variants"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "",
    "panel_section_id": "gatk",
    "version": "0.0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_apply_recalibration%2Fgatk_variant_apply_recalibration%2F0.0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_apply_recalibration/gatk_variant_apply_recalibration/0.0.4",
    "name": "Apply Variant Recalibration"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "",
    "panel_section_id": "gatk",
    "version": "0.0.5",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvariant_annotator%2Fgatk_variant_annotator%2F0.0.5",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/variant_annotator/gatk_variant_annotator/0.0.5",
    "name": "Variant Annotator"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "SNP and indel caller",
    "panel_section_id": "gatk",
    "version": "0.0.6",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Funified_genotyper%2Fgatk_unified_genotyper%2F0.0.6",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/unified_genotyper/gatk_unified_genotyper/0.0.6",
    "name": "Unified Genotyper"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "for use in local realignment",
    "panel_section_id": "gatk",
    "version": "0.0.4",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Frealigner_target_creator%2Fgatk_realigner_target_creator%2F0.0.4",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/realigner_target_creator/gatk_realigner_target_creator/0.0.4",
    "name": "Realigner Target Creator"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "from BAM files",
    "panel_section_id": "gatk",
    "version": "0.0.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fprint_reads%2Fgatk_print_reads%2F0.0.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/print_reads/gatk_print_reads/0.0.1",
    "name": "Print Reads"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "- perform local realignment",
    "panel_section_id": "gatk",
    "version": "0.0.6",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Findel_realigner%2Fgatk_indel_realigner%2F0.0.6",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/indel_realigner/gatk_indel_realigner/0.0.6",
    "name": "Indel Realigner"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "on BAM files",
    "panel_section_id": "gatk",
    "version": "0.0.2",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdepth_of_coverage%2Fgatk_depth_of_coverage%2F0.0.2",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/depth_of_coverage/gatk_depth_of_coverage/0.0.2",
    "name": "Depth of Coverage"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "on BAM files",
    "panel_section_id": "gatk",
    "version": "0.0.5",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcount_covariates%2Fgatk_count_covariates%2F0.0.5",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/count_covariates/gatk_count_covariates/0.0.5",
    "name": "Count Covariates"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "- draw plots",
    "panel_section_id": "gatk",
    "version": "0.0.5",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fanalyze_covariates%2Fgatk_analyze_covariates%2F0.0.5",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/analyze_covariates/gatk_analyze_covariates/0.0.5",
    "name": "Analyze Covariates"
  }, {
    "panel_section_name": "NGS: GATK Tools (beta)",
    "description": "on BAM files",
    "panel_section_id": "gatk",
    "version": "0.0.5",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftable_recalibration%2Fgatk_table_recalibration%2F0.0.5",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/table_recalibration/gatk_table_recalibration/0.0.5",
    "name": "Table Recalibration"
  }], "model_class": "ToolSection", "version": "", "id": "gatk", "name": "NGS: GATK Tools (beta)"
}, {
  "elems": [{
    "panel_section_name": "NGS: Picard (beta)",
    "description": "creates an unaligned BAM file",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FastqToSam%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_FastqToSam/1.56.0",
    "name": "FASTQ to BAM"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "creates a FASTQ file",
    "panel_section_id": "picard_beta",
    "version": "1.56.1",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_SamToFastq%2F1.56.1",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_SamToFastq/1.56.1",
    "name": "SAM to FASTQ"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_BamIndexStats%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_BamIndexStats/1.56.0",
    "name": "BAM Index Statistics"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FPicardASMetrics%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/PicardASMetrics/1.56.0",
    "name": "SAM/BAM Alignment Summary Metrics"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FPicardGCBiasMetrics%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/PicardGCBiasMetrics/1.56.0",
    "name": "SAM/BAM GC Bias Metrics"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FrgEstLibComp%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/rgEstLibComp/1.56.0",
    "name": "Estimate Library Complexity"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "for PAIRED data",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FPicardInsertSize%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/PicardInsertSize/1.56.0",
    "name": "Insertion size metrics"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "for targeted resequencing data",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FPicardHsMetrics%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/PicardHsMetrics/1.56.0",
    "name": "SAM/BAM Hybrid Selection Metrics"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ARRG%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_ARRG/1.56.0",
    "name": "Add or Replace Groups"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ReorderSam%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_ReorderSam/1.56.0",
    "name": "Reorder SAM/BAM"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ReplaceSamHeader%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_ReplaceSamHeader/1.56.0",
    "name": "Replace SAM/BAM Header"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "for paired data",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FrgPicFixMate%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/rgPicFixMate/1.56.0",
    "name": "Paired Read Mate Fixer"
  }, {
    "panel_section_name": "NGS: Picard (beta)",
    "description": "",
    "panel_section_id": "picard_beta",
    "version": "1.56.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2FrgPicardMarkDups%2F1.56.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/devteam/picard/rgPicardMarkDups/1.56.0",
    "name": "Mark Duplicate reads"
  }], "model_class": "ToolSection", "version": "", "id": "picard_beta", "name": "NGS: Picard (beta)"
}, {
  "elems": [{
    "panel_section_name": "RNA Structure Prediction",
    "description": "iteratively maps the raw reads of RNA structural data to the reference transcriptome",
    "panel_section_id": "rna_structure_prediction",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Ftyty%2Fstructurefold%2Fiterative_map_pipeline%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/tyty/structurefold/iterative_map_pipeline/1.0",
    "name": "Iterative Mapping"
  }, {
    "panel_section_name": "RNA Structure Prediction",
    "description": "derives the reverse transcriptase (RT) stop count on each nucleotide from a mapped file provided by the Iterative Mapping module",
    "panel_section_id": "rna_structure_prediction",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Ftyty%2Fstructurefold%2Fget_read_pipeline%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/tyty/structurefold/get_read_pipeline/1.0",
    "name": "Get RT Stop Counts"
  }, {
    "panel_section_name": "RNA Structure Prediction",
    "description": "calculates structural reactivity on each nucleotide based on RT stop counts from the Get RT Stop Counts module",
    "panel_section_id": "rna_structure_prediction",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Ftyty%2Fstructurefold%2Freact_cal_pipeline%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/tyty/structurefold/react_cal_pipeline/1.0",
    "name": "Reactivity Calculation"
  }, {
    "panel_section_name": "RNA Structure Prediction",
    "description": "predict RNA structures with or without experimental constraints from the Reactivity Calculation module",
    "panel_section_id": "rna_structure_prediction",
    "version": "1.0",
    "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Ftyty%2Fstructurefold%2Fpredict_pipeline%2F1.0",
    "target": "galaxy_main",
    "min_width": -1,
    "model_class": "Tool",
    "id": "toolshed.g2.bx.psu.edu/repos/tyty/structurefold/predict_pipeline/1.0",
    "name": "RNA Structure Prediction"
  }], "model_class": "ToolSection", "version": "", "id": "rna_structure_prediction", "name": "RNA Structure Prediction"
}];

var test = {
  "panel_section_name": "FASTA manipulation",
  "description": "converts tabular file to FASTA format",
  "panel_section_id": "fasta_manipulation",
  "version": "1.1.0",
  "link": "/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftabular_to_fasta%2Ftab2fasta%2F1.1.0",
  "target": "galaxy_main",
  "min_width": -1,
  "model_class": "Tool",
  "id": "toolshed.g2.bx.psu.edu/repos/devteam/tabular_to_fasta/tab2fasta/1.1.0"
}